sql/2019/10_SEO/10_15.sql

```diff
@@ -0,0 +1,20 @@
+#standardSQL
+
+# speed metrics (FCP, server response time)
+
+# speeds from lighthouse
+# speeds from Crux (more interesting?)
+# https://discuss.httparchive.org/t/measuring-cms-host-ttfb-in-crux/1676/1
+
+# todo: onload event? document html size? images size? soo many more metrics
+
+SELECT
+    COUNT(url) AS `total`,
+    COUNT(DISTINCT url) AS `distinct_total`,
+    AVG(SAFE_CAST(JSON_EXTRACT_SCALAR(report, '$.audits.time-to-first-byte.numericValue') as NUMERIC)) AS `ttfbAvgMs`,
```

Comment:
Let's use CrUX rather than Lighthouse since synthetic performance metrics are not necessarily representative of real user experiences.